webui/react/src/pages/ExperimentDetails/ExperimentTrials.tsx

```diff
@@ -149,10 +150,12 @@ const ExperimentTrials: React.FC<Props> = ({ experiment, pageRef }: Props) => {
       );
     };
 
-    const validationRenderer = (key: string) => {
+    const validationRenderer = (key: keyof TrialItem) => {
       return function renderer(_: string, record: TrialItem): React.ReactNode {
-        /* eslint-disable-next-line @typescript-eslint/no-explicit-any */
-        const value = getMetricValue((record as any)[key], metric);
+        const value = getMetricValue(
+          record[key] as { metrics?: Record<RecordKey, number> },
```

Comment:
If you look at `TrialItem` only some of the keys are objects. You need to start by narrowing down the type of `key` to just the valid keys.

Reply:
You marked this resolved but I don't see how it has been addressed.